python/cli/prelude_cli/views/build.py

```diff
@@ -41,23 +41,28 @@ def create_test(controller, name, unit, test, techniques, advisory):
             advisory=advisory
         )
 
-    if not test:
-        basename = f'{t["id"]}.go'
-        template = pkg_resources.read_text(templates, 'template.go')
+    def create_template(template, name):
+        utc_time = str(datetime.now(timezone.utc))
+        template = pkg_resources.read_text(templates, template)
```

Comment:
nit: will you call the input param something other than `template` just so we don't have the variable name reuse

Reply:
i switched up the name of the variable, instead.